openpype/plugins/publish/integrate_thumbnail_ayon.py

```diff
@@ -5,7 +5,19 @@
     pull into a scene.
 
     This one is used only as image describing content of published item and
-    shows up only in Loader in right column section.
+    shows up only in Loader or Server UI in right column section.
+
+    Possible sources of thumbnail paths (OR):
+    - context.data.get("thumbnailPath")
+    (instance must have "published_representations")
+    - instance.data.get("thumbnailSource")
```

Comment:
This should not be used https://github.com/ynput/OpenPype/pull/6063
```suggestion
```

Reply:
Replaced by this comment https://github.com/ynput/OpenPype/pull/6061/files#r1429866462